src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentRequestResourceIT.java

```diff
@@ -178,10 +178,10 @@ public void shouldReturn400IfFieldsInvalidSize() {
     }
 
     @Test
-    public void shouldReturn400IfFieldsInvalid() {
+    public void shouldReturn400IfFieldsInvalid() throws JsonProcessingException {
         String accountId = testGatewayAccount.getId().toString();
 
-        String postBody = gson.toJson(ImmutableMap.builder()
+        String postBody = new ObjectMapper().writeValueAsString(ImmutableMap.builder()
```

Comment:
Minor - For next PR, ObjectMapper is an expensive object to build every time, is test code so is fine but can we added in the Before

Reply:
sure!